test/test-helpers/common.js

```diff
@@ -938,6 +938,23 @@ export default function (repository, dbClient) {
                     });
             });
 
+            it('should return the maximum age', (done) => {
+
+                repository.Models.PersonModel
+                    .forge()
+                    .fetchJsonApi({
+                        fields: {
+                            person: ['max(age)']
```

Comment:
Why was this not working previously? Can you point me to your change the fixed it and why is was broken?